Sources/PubNubChatComponents/Controllers/ComponentViewController.swift

```diff
@@ -125,4 +127,22 @@ open class CollectionViewComponent: ComponentViewController  {
     collectionView.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 0).isActive = true
     collectionView.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: 0).isActive = true
   }
+  
+  open func configureGestureRecognizers() {
+    let longPressGesture = UILongPressGestureRecognizer(target: self, action: #selector(type(of: self).onLongPressGestureRecognized(gesture:)))
+    let swipeGestureRecognizer = UISwipeGestureRecognizer(target: self, action: #selector(type(of: self).onSwipeGestureRecognized(gesture:)))
```

Comment:
Do we need to add `type(of: self).` to selector?

Reply:
No, I can change it to use a class type instead